### DISCUSSION TITLE
Google Chrome Error code: Out of Memory

### DISCUSSION THREAD

**Discussion by samnittala** [cypress-io/cypress#21221] [General]
I am experiencing this error in Google Chrome Version 101.0.4951.41 (Official Build) (64-bit) after my tests runs for about 300 secs 

"Aw, Snap!"

**Comment by tit** [comment]
@samnittala try to add in your cypress.json file next value:
```
{
  "numTestsKeptInMemory": 1
}
```

https://docs.cypress.io/guides/references/configuration#Global
